### DISCUSSION TITLE
Multiline details for tasks

### DISCUSSION THREAD

**Discussion by a-delannoy** [u8slvn/hyperfocus#6] [Ideas]
Hey,

Great project ! What would you think of having the possibility to have multiline details associated with tasks that we could edit in kind of the same way git works.

For example, setting details on a task, hyperfocus would open a previously configured text editor to edit details in an easier way !

**Comment by u8slvn** [comment]
Hi,
Indeed, it's a great idea! We already talked about it in your last PR. I don't know how GitHub Projects feature works, but I'll try to write this within a ticket.

**Comment by u8slvn** [comment]
My bad, no need to use GitHub projects, I just created an issue. You can keep track of it here: #8

**Comment by a-delannoy** [comment]
Great, let me dive into this !
